WordPress/gutenberg
# [Block Library - Site Logo]: Return nothing if logo is not set

## Description
Currently, the site logo block renders an empty div if the theme has support for a custom logo but the user has not set one. 

This PR adjusts the render callback to return nothing if the custom logo has not been set. 

## How has this been tested?
- Activate a theme with custom logo support
- Remove the logo if one has been set
- Verify that no markup is generated by the block  

## Types of changes
Bug fix

## Checklist:
